src/platform/mhv/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@department-of-veterans-affairs/mhv",
```

Comment:
Don't you have to add this to the babel.config.json file too?

Reply:
It's unclear to me based on [the bullet point from the Platform docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/guide-for-engineers#WorktreeandWorkspaceSetupGuide-src/platformcodeworkspaces:):

> When importing **nested** modules from a yarn workspace (i.e. modules inside a folder inside the workspace) ESlint will not be able to resolve the module specified by the new yarn workspace syntax unless an alias for the module is added to the root level `babel.config.json`.

It might be something that needs to be done on a per-file basis. It seems that apps do map their exports files, so I guess I should add that to start